samples/msal-node-samples/auth-code-distributed-cache/src/AuthProvider.ts

```diff
@@ -102,80 +114,136 @@ export class AuthProvider {
         return { authCodeUrl, state };
     }
 
-    async getTokenInteractive(tokenRequest: AuthorizationCodeRequest, authCodePayLoad: AuthorizationCodePayload, sessionId: string): Promise<AuthenticationResult | null> {
-        if (tokenRequest.authority && tokenRequest.authority !== this.msalConfig.auth.authority) {
+    async getTokenInteractive(
+        tokenRequest: AuthorizationCodeRequest,
+        authCodePayLoad: AuthorizationCodePayload,
+        sessionId: string
+    ): Promise<AuthenticationResult | null> {
+        if (
+            tokenRequest.authority &&
+            tokenRequest.authority !== this.msalConfig.auth.authority
+        ) {
             this.msalConfig.auth.authority = tokenRequest.authority;
-            this.msalConfig = await AuthProvider.getMetadata(this.msalConfig, this.cacheClient);
+            this.msalConfig = await AuthProvider.getMetadata(
+                this.msalConfig,
+                this.cacheClient
+            );
         }
 
         const msalInstance = this.getMsalInstance(sessionId);
         let tokenResponse = null;
 
         performance.mark("acquireTokenByCode-start");
         await msalInstance.getTokenCache().getAllAccounts(); // required for triggering beforeCacheACcess
-        tokenResponse = await msalInstance.acquireTokenByCode(tokenRequest, authCodePayLoad);
+        tokenResponse = await msalInstance.acquireTokenByCode(
+            tokenRequest,
+            authCodePayLoad
+        );
         performance.mark("acquireTokenByCode-end");
-        performance.measure("acquireTokenByCode", "acquireTokenByCode-start", "acquireTokenByCode-end");
+        performance.measure(
+            "acquireTokenByCode",
+            "acquireTokenByCode-start",
+            "acquireTokenByCode-end"
+        );
 
         return tokenResponse;
     }
 
-    async getTokenSilent(silentRequest: SilentFlowRequest, sessionId: string): Promise<AuthenticationResult | null> {
+    async getTokenSilent(
+        silentRequest: SilentFlowRequest,
+        sessionId: string
+    ): Promise<AuthenticationResult | null> {
         const msalInstance = this.getMsalInstance(sessionId);
         let tokenResponse = null;
         performance.mark("acquireTokenSilent-start");
         await msalInstance.getTokenCache().getAllAccounts(); // required for triggering beforeCacheACcess
         tokenResponse = await msalInstance.acquireTokenSilent(silentRequest);
         performance.mark("acquireTokenSilent-end");
-        performance.measure("acquireTokenSilent", "acquireTokenSilent-start", "acquireTokenSilent-end");
+        performance.measure(
+            "acquireTokenSilent",
+            "acquireTokenSilent-start",
+            "acquireTokenSilent-end"
+        );
 
         return tokenResponse;
     }
 
-    private static async getMetadata(msalConfig: Configuration, cacheClient: RedisClientType): Promise<Configuration> {
+    private static async getMetadata(
```

Comment:
In the future, I recommend that you create separate PRs for refactoring vs actual changes. This is a huge PR and I am not able to focus on actual changes.

Reply:
I'll make a separate PR for just the formatting changes. It's exponentially inflating this PR. Nothing I can do about it, they happen when I save a file. We've never linted the msal-node samples before.

Reply:
Done. This PR is much easier to read now. [Linted/Formatted all msal-node samples](https://github.com/AzureAD/microsoft-authentication-library-for-js/pull/7450)